code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -511,9 +513,25 @@ This is here to make the tiles around the station mininuke change when it's arme
 	else
 		GLOB.poi_list |= src
 		tell_the_admins = TRUE
+		START_PROCESSING(SSobj, src)
 
 	set_stationloving(TRUE, inform_admins=tell_the_admins)
 
+/obj/item/disk/nuclear/process()
+	if(fake)
+		STOP_PROCESSING(SSobj, src)
+		return
+	var/turf/newturf = get_turf(src)
+	if(istype(newturf) && lastlocation == newturf)
+		if(loneop_event_weight_increase_chance > 5 && prob(loneop_event_weight_increase_chance - 5))
+			var/datum/round_event_control/loneop = locate(/datum/round_event_control/operative) in SSevents.control
```

Comment:
`var/datum/round_event_control/loneop/loneop` to avoid future potential mishaps :P